recipes/shorah/1.1.3/meta.yaml

```diff
@@ -11,7 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 6
+  number: 7
```

Comment:
If you prefere you can use a build.sh file or specify multiple commands in a list

Reply:
So we have a guarantee that the command will be ran sequencially?

Okay, will do. Thanks for the suggestion.